core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.spec.ts

```diff
@@ -163,6 +189,26 @@ describe('Story node editor directive', function() {
       '/skill_editor/skill_1');
   });
 
+  it('should fetch the descriptions for prerequisite skills', function() {
+    spyOn($scope, 'getPrerequisiteSkillIds').and.returnValues(['1', '0']);
+
+    $scope.getPrerequisiteSkillsDescription();
+    $rootScope.$apply();
+    expect($scope.skillIdToSummaryMap).toEqual({1: 'test'});
+
+    $scope.getPrerequisiteSkillsDescription();
+    $rootScope.$apply();
+    expect($scope.skillIdToSummaryMap).toEqual({});
```

Comment:
Why is this empty?

Reply:
It was left in the code by mistake, I have removed it now.